Rocket.Chat/Controllers/Preferences/ChangeAppIcon/ChangeAppIconCell.swift

```diff
@@ -10,23 +10,30 @@ import UIKit
 
 final class ChangeAppIconCell: UICollectionViewCell {
 
-    @IBOutlet private weak var iconImageView: UIImageView!
+    @IBOutlet private weak var iconImageView: UIImageView! {
+        didSet {
+            iconImageView.isAccessibilityElement = true
+        }
+    }
     @IBOutlet private weak var checkImageView: UIImageView!
     @IBOutlet private weak var checkImageViewBackground: UIView!
 
-    func setIcon(name: String, selected: Bool) {
-        iconImageView.image = UIImage(named: name)
+    func setIcon(name: (String, String), selected: Bool) {
+        iconImageView.image = UIImage(named: name.0)
+        iconImageView.accessibilityLabel = VOLocalizedString(name.1)
 
         if selected {
             iconImageView.layer.borderColor = UIColor.RCBlue().cgColor
             iconImageView.layer.borderWidth = 3
+            iconImageView.accessibilityTraits = .selected
 
             checkImageView.image = checkImageView.image?.imageWithTint(UIColor.RCBlue())
             checkImageView.isHidden = false
             checkImageViewBackground.isHidden = false
         } else {
             iconImageView.layer.borderColor = UIColor.RCLightGray().cgColor
             iconImageView.layer.borderWidth = 1
+            iconImageView.accessibilityTraits = .image
```

Comment:
@rudrankriyam I don't know if the icons should present themselves as an image. Maybe `.none` seems like a better fit. Lemme know how you feel about this.

Reply:
.image was more precise, but as it is clear that it is an icon, .none fits better, especially with .selected. 
So, it is now -> selected white icon and white icon with .none instead of selected white icon and white icon image, with .image

Made the changes! @Sameesunkaria